src/App.jsx

```diff
@@ -26,20 +27,21 @@ export const alex = {
 export const App = () => (
   <div className="App">
     <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
+      <Person
+        person={misha}
+      />
```

Comment:
just wrap if you have more than 2 params || length more than 80
```suggestion
      <Person person={misha}/>
```